packages/fast-components-styles-msft/src/tree-view-item/index.ts

```diff
@@ -0,0 +1,228 @@
+import { ComponentStyles } from "@microsoft/fast-jss-manager";
+import { TreeViewItemClassNameContract } from "@microsoft/fast-components-class-name-contracts-msft";
+import { DesignSystem } from "../design-system";
+import {
+    add,
+    applyFocusVisible,
+    directionSwitch,
+    divide,
+    format,
+    multiply,
+    subtract,
+    toPx,
+} from "@microsoft/fast-jss-utilities";
+import {
+    accentForegroundRest,
+    neutralFillStealthActive,
+    neutralFillStealthHover,
+    neutralFillStealthRest,
+    neutralFillStealthSelected,
+    neutralFocus,
+    neutralForegroundRest,
+} from "../utilities/color";
+import { heightNumber } from "../utilities/density";
+import { designUnit, focusOutlineWidth } from "../utilities/design-system";
+import {
+    HighContrastColor,
+    highContrastDoubleFocus,
+    highContrastOptOutProperty,
+    highContrastSelected,
+    highContrastSelectedBackground,
+    highContrastSelector,
+} from "../utilities/high-contrast";
+import { applyCornerRadius, applyFocusPlaceholderBorder } from "../utilities/border";
+import { applyCursorPointer } from "../utilities/cursor";
+import {
+    glyphSize,
+    glyphSizeNumber,
+    height,
+    horizontalSpacing,
+} from "../utilities/density";
+import {
+    applyScaledLineHeight,
+    applyScaledTypeRamp,
+    getScaledLineHeight,
+} from "../utilities/typography";
+import { applyFontWeightNormal } from "../utilities/fonts";
+
+const nestedNodeOffest: (designSystem?: DesignSystem) => string = toPx(
```

Comment:
Seems like this needs to be typed as DesignSystemResolver<string>. If the designSystem is optional, the default values will be used, so any design system changes won't apply.